tests/Integration/Database/EloquentBelongsToManyTest.php

```diff
@@ -791,53 +795,54 @@ public function testNoTouchingHappensIfNotConfigured()
         $this->assertNotSame('2017-10-10 10:10:10', $tag->fresh()->updated_at->toDateTimeString());
     }
 
-    /** @group SkipMSSQL */
     public function testCanRetrieveRelatedIds()
     {
         $post = Post::create(['title' => Str::random()]);
 
         DB::table('tags')->insert([
-            ['id' => 200, 'name' => 'excluded'],
-            ['id' => 300, 'name' => Str::random()],
+            ['name' => 'excluded'],
+            ['name' => Str::random()],
         ]);
 
         DB::table('posts_tags')->insert([
-            ['post_id' => $post->id, 'tag_id' => 200, 'flag' => ''],
-            ['post_id' => $post->id, 'tag_id' => 300, 'flag' => 'exclude'],
-            ['post_id' => $post->id, 'tag_id' => 400, 'flag' => ''],
+            ['post_id' => $post->id, 'tag_id' => 1, 'flag' => ''],
+            ['post_id' => $post->id, 'tag_id' => 2, 'flag' => 'exclude'],
+            ['post_id' => $post->id, 'tag_id' => 3, 'flag' => ''],
         ]);
 
-        $this->assertEquals([200, 400], $post->tags()->allRelatedIds()->toArray());
+        $this->assertEquals([1, 3], $post->tags()->allRelatedIds()->toArray());
```

Comment:
It's not possible to insert a primary key in a table in MSSQL without enabling that explicitly. Since it's not the main point of this test and the tests below, I've removed all explicit primary identifier values when creating DB entries and opted to instead check their sensible incremental defaults.